packages/typespec-client-generator-core/generated-defs/Azure.ClientGenerator.Core.ts

```diff
@@ -511,18 +511,19 @@ export type ParamAliasDecorator = (
  * By default, the client namespace for them will follow the TypeSpec namespace.
  *
  * @param rename The rename you want applied to the object
- * @param scope The language scope you want this decorator to apply to. If not specified, will apply to all language emitters
+ * @param scope The language scope you want this decorator to apply to. If not specified, will apply to all language emitters.
+ * You can use "!" to specify negation such as "!(java, python)" or "!java, !python".
  * @example
  * ```typespec
  * @clientNamespace("ContosoClient")
  * namespace Contoso;
  * ```
  * @example
  * ```typespec
- * @clientName("ContosoJava", "java")
- * @clientName("ContosoPython", "python")
- * @clientName("ContosoCSharp", "csharp")
- * @clientName("ContosoJavascript", "javascript")
+ * @clientNamespace("ContosoJava", "java")
```

Comment:
Why is this PR including things about clientNamespace?

Reply:
i just noticed this nit for the `@clientNamespace` doc and fixed it.

Reply:
@lmazuel if no concern, could you also help to approve this pr?

Reply:
@tadelesh approved!